csrc/transformer/dropout_kernels.cu

```diff
@@ -13,17 +15,17 @@ __global__ void dropout_kernel(const int N,
     curandStatePhilox4_32_10_t state;
     curand_init(seed.first, idx, seed.second, &state);
 
-    CUDA_1D_KERNEL_LOOP(j, N / 4)
+    CUDA_1D_KERNEL_LOOP(j, N / unroll_factor)

```

Comment:
What if N is not divisible by the unroll factor? Should we take care of the boundary condition?

Reply:
yes, I can do that, however, I need to access the last part that is indivisible one by one. I will add this part to the code too.